ddp-workspace/projects/ddp-sdk/src/lib/components/cookies/cookiesBanner/cookiesBanner.component.ts

```diff
@@ -0,0 +1,69 @@
+import { Component, Inject } from '@angular/core';
+import { CookiesManagementService } from '../../../services/cookiesManagement.service';
+import { MatDialog } from '@angular/material/dialog';
+import { ConfigurationService } from '../../../services/configuration.service';
+import { NoopScrollStrategy } from '@angular/cdk/overlay';
+import { CookiesPreferencesModalComponent } from '../cookiesPreferencesModal/cookiesPreferencesModal.component';
+import { PrivacyPolicyModalComponent } from '../privacyPolicy/privacyPolicyModal.component';
+
+@Component({
+  selector: 'ddp-cookies-banner',
+  template: `
+  <div class="cookiesBanner">
+    <div class="cookiesBanner__text">
+      <span translate>SDK.CookiesBanner.Message_First</span>
```

Comment:
Theoretically, it is likely that other studies will want to use this feature.  Dividing it into Message_First, Message_Second, etc. is not entirely intuitive.  Instead, maybe combine Message_First and Message_Second (and call them something that indicates this is the portion of the main text that comes before the privacy policy link), keep CookiesBanner.Policy, and then rename Message_Third something that indicates it is the portion of the main text that comes after the privacy policy link.

Reply:
Done in 527818e